ts/Series/Organization/OrganizationDataLabelOptions.d.ts

```diff
@@ -47,6 +47,7 @@ export interface OrganizationDataLabelFormatterContext extends SankeyDataLabelFo
 
 export interface OrganizationDataLabelOptions extends SankeyDataLabelOptions {
     nodeFormatter?: OrganizationDataLabelsFormatterCallbackFunction;
+    linkFormat?: string;
```

Comment:
how about `linkFormatter`? Shouldn't it also be included?

Reply:
Also, `linkTextPath`. (For the pattern, see `TreegraphDataLabelOptions` in `TreegraphSeriesOptions.d.ts`)

Reply:
You're right, I've included `linkFormatter`. But should I include the `linkTextPath` if it exists on `SankeyDataLabelOptions`?

Reply:
Right, Not necessary! Thanks!